homeassistant/components/trello/coordinator.py

```diff
@@ -0,0 +1,69 @@
+"""Data update coordinator for the Steam integration."""
```

Comment:
```suggestion
"""Data update coordinator for the Trello integration."""
```

Reply:
Busted :( Nice catch.

Reply:
In dutch we have a saying "Better stolen well than invented poorly"